coralproject/talk
# [CORL-2279]: Add list of positive reactions to rejected comments to comment cards

## What does this PR do?

These changes add a `Reactions` tab to the `Details` dropdown on comment cards when a comment has been reacted to. In this tab, the usernames of the users who have reacted to the comment are shown. When their usernames are clicked on, their user history drawer opens up to show more information about them. These changes are for https://vmproduct.atlassian.net/browse/CORL-2279.

## What changes to the GraphQL/Database Schema does this PR introduce?

None

## How do I test this PR?

Make some users react to a comment. Go to that comment's card in `Moderate` and click `Details`. See that there is a `Reactions` tab and that it is the second tab after `Info`. Click on the `Reactions` tab. See that this is when the query is run to get the reaction data. See that the usernames for the reactions are displayed. If you click on a username, the user history drawer for that user opens. If there are more users than 20 who have reacted to a comment, a `Load More` button will appear to load more.

If a comment has no reactions, if you click `Details` on its comment card, it should not have a `Reactions` tab option.
 
## How do we deploy this PR?
